src/ferry_planner/schedule.py

```diff
@@ -48,6 +53,17 @@ def __call__(
     ) -> FerrySchedule | None: ...
 
 
+class HtmlParseResult:
+    redirect_url: str | None = None
+    sailings: Sequence[FerrySailing] | None = None
```

Comment:
Better to make this required and create the `HtmlParseResult` object at the end of the function when returning